Builder_/Program.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+
+namespace builder_
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            // Client Code
+            var whitePizzaBuilder = new WhitePizzaBuilder();
+            var director = new Director(whitePizzaBuilder);
+            director.makePizza();
+            Console.WriteLine(whitePizzaBuilder.GetPizza().DelieveredPizza());
+
+            //Bonus 1: the client plays the Director role and interact with PizzaBuilder directly.
+            var wheatPB = new WheatPizzaBuilder();
+            wheatPB
+            .BuildDough()
+            .BuildTopping()
+            .BuildSauce();
+            Console.WriteLine(wheatPB.GetPizza().DelieveredPizza());
+
+            //Bonus 2: Accept entres from client to choose how the pizza is going to be cooked
```

Comment:
Good job entering the fields for the `Pizza`, but as written in the requirements you should ask the user to enter the input not to be hard coded.